src/app/mail/mail-list/mail-folder/generic-folder/generic-folder.component.ts

```diff
@@ -97,6 +97,25 @@ export class GenericFolderComponent implements OnInit, OnDestroy, OnChanges {
         }
       });
 
+    this.activatedRoute.paramMap.pipe(takeUntil(this.destroyed$))
+      .subscribe((paramsMap: any) => {
```

Comment:
Why not subscribe on `params` instead of `paramMap`?

Reply:
no specific reason